### ISSUE TITLE
#Big_problem new update All app bank error I make #magiskHide? But still Error?

### ISSUE THREAD

**Issue by NaderKabir** [topjohnwu/Magisk#3853]
![Screenshot_20210214-215529](https://user-images.githubusercontent.com/25882967/107878892-98e16400-6f10-11eb-962a-3497ff88a61a.jpg)
![Screenshot_20210214-215358](https://user-images.githubusercontent.com/25882967/107878895-9d0d8180-6f10-11eb-80c3-ca67352531b2.jpg)
![Screenshot_20210214-215342](https://user-images.githubusercontent.com/25882967/107878902-a0a10880-6f10-11eb-97e7-af5b3bac55ed.jpg)

## READ BEFORE OPENING ISSUES

All bug reports require you to **USE CANARY BUILDS**. Please include the version name and version code in the bug report.

If you experience a bootloop, attach a `dmesg` (kernel logs) when the device refuse to boot. This may very likely require a custom kernel on some devices as `last_kmsg` or `pstore ramoops` are usually not enabled by default. In addition, please also upload the result of `cat /proc/mounts` when your device is working correctly **WITHOUT ROOT**.

If you experience issues during installation, in recovery, upload the recovery logs, or in Magisk Manager, upload the install logs. Please also upload the `boot.img` or `recovery.img` that you are using for patching.

If you experience a crash of Magisk Manager, dump the full `logcat` **when the crash happens**. **DO NOT** upload `magisk.log`.

If you experience other issues related to Magisk, upload `magisk.log`, and preferably also include a boot `logcat` (start dumping `logcat` when the device boots up)

**DO NOT** open issues regarding root detection.

**DO NOT** ask for instructions.

**DO NOT** report issues if you have any modules installed.

Without following the rules above, your issue will be closed without explanation.

**Comment by Haus3master** [comment]
The apps might be using new methods for root detection, and, I recommend you close this issue, since Magisk hide issues can't be solved most of the time unless you bring new methods every time for detection changes, which isn't that easy...

**Comment by Mspy1** [comment]
Have you read this before opening an issue?
![image](https://user-images.githubusercontent.com/33182973/107881754-bbff1a00-6ef6-11eb-8f6d-83da00692b4b.png)

**Comment by Jon8RFC** [comment]
@NaderKabir 
They both work for me.  At first, I got the same problems as you, but I was able to make BOTH work with these steps:
Run both apps at least once and get the "root detected" error.
Make sure you're updated to the latest magisk 21410.
Hide magisk itself and rename it through the magisk app.
Hide the apps in magisk hide and tick both options under Jenius since enabling only selects one.
Force stop and clear data for both apps, start them again.
